server/utils/matchUtils.test.ts

```diff
@@ -37,4 +47,17 @@ describe('matchUtils', () => {
       expect(mapApiCharacteristicForUi(pipe)).toBe('<li>PIPE</li>')
     })
   })
+
+  describe('summaryCardsRow', () => {
+    it('calls the correct row functions', () => {
+      const bedSearchResult = bedSearchResultFactory.build()
+      expect(summaryCardRows(bedSearchResult.results[0])).toEqual([
+        townRow(bedSearchResult.results[0]),
```

Comment:
Should we test the individual row methods here?

Reply:
Our test coverage checker doesn't count them as functions however I think it would be worthwhile to test that the complete row is as we expect - will add it in